primeui-tester/src/test/java/com/axonivy/ivy/webtest/primeui/TestPrimeUi.java

```diff
@@ -10,27 +10,35 @@
 import java.util.Arrays;
 
 import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 
-import com.axonivy.ivy.supplements.IvySelenide;
-import com.axonivy.ivy.supplements.primeui.tester.ShowcaseUtil.Showcase;
-import com.axonivy.ivy.supplements.primeui.tester.widget.Accordion;
-import com.axonivy.ivy.supplements.primeui.tester.widget.Dialog;
-import com.axonivy.ivy.supplements.primeui.tester.widget.SelectBooleanCheckbox;
-import com.axonivy.ivy.supplements.primeui.tester.widget.SelectManyCheckbox;
-import com.axonivy.ivy.supplements.primeui.tester.widget.SelectOneMenu;
-import com.axonivy.ivy.supplements.primeui.tester.widget.SelectOneRadio;
-import com.axonivy.ivy.supplements.primeui.tester.widget.Table;
+import com.axonivy.ivy.webtest.primeui.ShowcaseUtil.Showcase;
+import com.axonivy.ivy.webtest.primeui.widget.Accordion;
+import com.axonivy.ivy.webtest.primeui.widget.Dialog;
+import com.axonivy.ivy.webtest.primeui.widget.SelectBooleanCheckbox;
+import com.axonivy.ivy.webtest.primeui.widget.SelectManyCheckbox;
+import com.axonivy.ivy.webtest.primeui.widget.SelectOneMenu;
+import com.axonivy.ivy.webtest.primeui.widget.SelectOneRadio;
+import com.axonivy.ivy.webtest.primeui.widget.Table;
+import com.codeborne.selenide.Configuration;
 
 /**
  * Class to test PrimeUi. Tests on the official Primefaces Showcase.
  */
-@IvySelenide
 public class TestPrimeUi
 {
   
+  @BeforeAll
```

Comment:
Cool change: doesn't look as if much of the infrastructure form @IvySeleniDE has been used in here. So this proves that we are on the right track by splitting these aritfacts.